armi/physics/neutronics/fissionProductModel/lumpedFissionProduct.py

```diff
@@ -259,10 +259,10 @@ def getLanthanideFraction(self):
 
     def printDensities(self, lfpDens):
         """Print densities of nuclides given a LFP density. """
-        nucs = self.keys()
+        nucs = list(self.keys())
         nucs.sort()
         for n in nucs:
```

Comment:
I think if you just make this `for n in sorted(self.keys()):` that would also work. But huh TIL that sorted takes more memory (because it makes an additional list apparently) https://blog.usejournal.com/list-sort-vs-sorted-list-aab92c00e17

Reply:
I'm going to make this change, but I have to force push because I rebased the branch onto the updated master, so the comment might go away.